protos/node/node.proto

```diff
@@ -128,7 +132,9 @@ message ContractStat {
 }
 
 message Contracts {
+  option (gogoproto.json_no_omit_empty) = true;
```

Comment:
suggest just use guard's contractMeta here,  as that contract should contains all information for the contract, and in the future, renter/host can use the signature of that contract to prove the contract's existence.

Reply:
This pr has nothing to do with the creation of contracts, it's simply a modification of marshal types. Can you mention the original design of not using contractMeta directly? @wxue

Reply:
@JustinGaoF That's a good suggestion, but the current contractMeta in Guard is not "meta" enough -- meaning, the contract here refers per file, not per shard, and this is mainly for UI/UX feed: https://docs.google.com/document/d/1OYMX4T-s_UexLooyo4Ldhj0rpN8Mh9DvurenLFTy4Zg/edit#heading=h.7w6f8xmxl4vj. 

Do you think we should make a reusable contractMeta with the most common basic info in it? - So node proto can also use it.
Will that create a big workload for the guard(/escrow)?

Reply:
I suggest doing a change later, perhaps when renter UI/UX design is finalized, that should help us see how "meta" the contracts message should get.

Reply:
sounds good

Reply:
+1